examples/access-optional-json-elements/access_optional_json_elements.md

```diff
@@ -0,0 +1,14 @@
+# Acessing optional JSON elements
+
+If there is no prior knowledge of the availability of a particular field, optional field access(?.) can be used on `json` value. If the particular field is not available, it will return `nil`.
```

Comment:
```suggestion
If there is no prior knowledge of the availability of a particular field, optional field access (`?.`) can be used on the `json` value. If the particular field is not available, it will return `nil`.
```